### PR TITLE
hardcode version for paul

### LINKED ISSUE #1347: PAUL's links don't work with version pages
**Describe the solution you'd like**
The [Paul Yaml](https://github.com/external-secrets/external-secrets/blob/main/.github/PAUL.yaml) file links to non version pages. Ex. `https://www.external-secrets.io/contributing-devguide/`

It appears this has been moved to `https://www.external-secrets.io/<version>/contributing-devguide`

**What is the added value?**
PR contributions become easier to click and find the proper documentation.

**Give us examples of the outcome**

Ideally there would be some process that injects the current version into paul. Maybe some sort of `awk|sed` process that when a version release occurs gets updated.

### PR DESCRIPTION
**Pull request by danquack**
Closes #1347 

Temp solution to at least make pauls comments clickable.

### CODE REVIEW

**Comment by gusfcarvalho** [comment]
Hi @danquack ! Thank you for your contribution! Setting up the url for the `latest` should take care of new releases (instead of fixing it to v0.5.7)

**Comment by gusfcarvalho** [comment]
(note: this is why [check-diff](https://github.com/external-secrets/external-secrets/runs/7298479992?check_suite_focus=true) is failing.

File: .github/PAUL.yaml [hunk 1]
```diff
@@ -42,8 +42,8 @@ pull_requests:
     Greetings!
     Thank you for contributing to this project!
     If this is your first time contributing, please make
-    sure to read the [Developer](https://www.external-secrets.io/contributing-devguide/) and [Contributing Process](https://www.external-secrets.io/contributing-process/) guides.
-    Please also mind and follow our [Code of Conduct](https://www.external-secrets.io/contributing-coc/).
+    sure to read the [Developer](https://www.external-secrets.io/v0.5.7/contributing-devguide/) and [Contributing Process](https://www.external-secrets.io/v0.5.7/contributing-process/) guides.
```
Comment (gusfcarvalho) [inline .github/PAUL.yaml:45]:
```suggestion
    sure to read the [Developer](https://www.external-secrets.io/latest/contributing-devguide/) and [Contributing Process](https://www.external-secrets.io/latest/contributing-process/) guides.
```

**Comment by gusfcarvalho** [comment]
Oh, it seems we've missed one! Sorry about that!

**Comment by gusfcarvalho** [comment]
@danquack it seems your commit came with a wrong signature! If you did `make docs`, there is a bug where it sets your git name and email to "Docs <docs.external-secrets.io>". Can you reset it and force push your original commit? Thanks ! :smile:

**Comment by danquack** [comment]
> @danquack it seems your commit came with a wrong signature! If you did `make docs`, there is a bug where it sets your git name and email to "Docs <docs.external-secrets.io>". Can you reset it and force push your original commit? Thanks ! 😄

just squashed and force pushed. Try that!

**Pull request merged**